test/parallel/test-process-load-env-file.js

```diff
@@ -48,10 +49,20 @@ describe('process.loadEnvFile()', () => {
     }, { code: 'ENOENT', syscall: 'open', path: missingEnvFile });
   });
 
+  // The whole chdir flow here is to address a case where a developer
+  // has a .env in the worktree which is probably in the global .gitignore.
+  // In that case this test would fail. To avoid confusion, chdir to lib will
+  // make this way less likely to happen. Probably a temporary directory would
+  // be the best choice but given how edge this case is this is fine.
   it('should throw when `.env` does not exist', async () => {
+    const originalCwd = process.cwd();
+    process.chdir(join(originalCwd, 'lib'));
+
     assert.throws(() => {
       process.loadEnvFile();
     }, { code: 'ENOENT', syscall: 'open', path: '.env' });
+
+    process.chdir(originalCwd);
```

Comment:
Can you put this in beforeEach/aafterEach or at least in a a try/finally so if the test fails it doesn't cause unrelated failures?

Reply:
Good point. I'll go for a try-finally tomorrow morning!

Reply:
@benjamingr Fixed. What about now?